internal/runbits/requirements/requirements.go

```diff
@@ -275,10 +280,18 @@ func (r *RequirementOperation) ExecuteRequirementOperation(ts *time.Time, requir
 	return nil
 }
 
+type ResolveNamespaceError struct {
+	*locale.LocalizedError
+	Name string
+}
+
 func (r *RequirementOperation) resolveNamespaces(ts *time.Time, requirements ...*Requirement) error {
 	for _, requirement := range requirements {
 		if err := r.resolveNamespace(ts, requirement); err != nil {
-			return errs.Wrap(err, "Could not resolve namespace")
+			return &ResolveNamespaceError{
+				locale.WrapError(err, "err_resolve_namespace", "Could not resolve namespace for requirement: {{.Name}}"),
```

Comment:
I think you can simplify this to `&ResolveNamespaceError{Name: requirement.Name}` because you're not using this error message; you're constructing a new one. Regardless, I don't think we're supposed to use `locale.WrapError()` in new code anymore as long as rationalizers are present.

Reply:
Yeah, I wasn't quite sure about that. I'l still embedded the error interface as we need to satisfy that.